code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -247,6 +247,7 @@
 			if(!disassembled)
 				stored.obj_integrity = stored.max_integrity * 0.5
 			transfer_fingerprints_to(stored)
+			. = stored
```

Comment:
Could you add some documentation that mentions wrench_act/deconstruct returning the pipe they touched? Aside from that this looks great

Reply:
@LemonInTheDark I added some doc.